### PR TITLE
Updated privacy policy 

### PR DESCRIPTION
**Pull request by UnrealApex**
Made minor changes.

### CODE REVIEW

File: static/privacy-policy.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
```
Comment (UnrealApex) [inline static/privacy-policy.html:4]: Visual Studio Code was doing some weird formatting using its default formatter, if possible could you format it correctly.

Comment (UnrealApex) [reply static/privacy-policy.html:4]: I have Allow edits by maintainers enabled

**Comment by Miodec** [comment]
All i see is whitespace changes. I think your prettier is messed up because it should be using 2 spaces but everything is now 4 spaces.

**Comment by UnrealApex** [comment]
I ran `npx prettier --write .` on the codebase, what do you think? I also found I small bug I had made where I made a closing attribute tag like this `<a/>`

**Comment by Miodec** [comment]
Prettier just runs on git commits. You dont need to install an extension or run it manually. You just need to run npm i and it will run by itself.

**Comment by UnrealApex** [comment]
`npm i` correct ?

**Comment by Miodec** [comment]
Yeah, then if you make a commit - it will run prettier.

**Comment by UnrealApex** [comment]
When I ran the command, there were no changes, but npm did add two files called `package-lock.json`

**Comment by Miodec** [comment]
Send me the changes in the privacy policy because I don't want to merge all the whitespace changes I feel like it's not done correctly. (running pretty independently instead of automatically on commit)

**Comment by UnrealApex** [comment]
here are the changes: 
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Monkeytype</title>
    <!-- <link rel="stylesheet" href="css/fa.css" /> -->
    <link rel="stylesheet" href="css/balloon.css" />
    <link rel="stylesheet" href="css/style.css" />
    <link rel="stylesheet" href="themes/serika_dark.css" id="currentTheme" />
    <link rel="stylesheet" href="" id="funBoxTheme" />
    <link id="favicon" rel="shortcut icon" href="images/fav.png" />
    <link rel="shortcut icon" href="images/fav.png" />
    <meta name="name" content="Monkeytype" />
    <meta name="image" content="https://monkeytype.com/mtsocial.png" />
    <meta
      name="description"
      content="A minimalistic, customisable typing website. Test yourself in various modes, track your progress and improve your typing speed."
    />
    <meta
      name="keywords"
      content="typing, test, typing-test, typing test, monkey-type, monkeytype, monkey type, monkey-types, monkeytypes, monkey types, types, monkey, type, miodec, wpm, words per minute, typing website, minimalistic, custom typing test, customizable, customisable, themes, random words, smooth caret, smooth, new, new typing site, new typing website, minimalist typing website, minimalistic typing website, minimalist typing test"
    />
    <meta name="author" content="Miodec" />
    <meta property="og:title" content="Monkeytype" />
    <meta property="og:url" content="https://monkeytype.com/" />
    <meta property="og:type" content="website" />
    <meta
      property="og:description"
      content="A minimalistic, customisable typing website. Test yourself in various modes, track your progress and improve your typing speed."
    />
    <meta property="og:image" content="https://monkeytype.com/mtsocial.png" />
    <meta name="theme-color" content="#e2b714" id="metaThemeColor" />
    <meta name="twitter:title" content="Monkeytype" />
    <meta name="twitter:image" content="https://monkeytype.com/mtsocial.png" />
    <meta name="twitter:card" content="summary_large_image" />
    <style>
      #top {
        font-size: 2.5rem;
      }

      #middle {
        color: var(--text-color);
      }

      h1 {
        font-weight: unset;
        color: var(--main-color);
        font-size: 2rem;
        margin-top: 3rem;
      }

      body {
        justify-content: center;
        display: flex;
      }
    </style>
  </head>

  <body>
    <div id="centerContent">
      <div id="top">
        <div class="logo">
          <div class="top">monkey see</div>
          <div class="bottom">
            <a href="/ " style="text-decoration: none; color: inherit">
              monkeytype
            </a>
            <span style="color: var(--text-color)">Privacy Policy</span>
          </div>
        </div>
      </div>
      <div id="middle">
        <p>
          This privacy policy explains how Monkeytype uses the data we collect
          from you when you use our website.
          <b>
            <u>
              IT IS IMPORTANT THAT YOU THOROUGHLY READ THIS POLICY BEFORE
              PROCEEDING ANY FURTHER.
            </u>
          </b>
        </p>

        <p>Table of Contents:</p>
        <ul>
          <li>What data do we collect?</li>
          <li>How do we collect your data?</li>
          <li>How will we use your data?</li>
          <li>How do we store your data?</li>
          <li>What are your data protection rights?</li>
          <li>What are cookies?</li>
          <li>How do we use cookies?</li>
          <li>What types of cookies do we use?</li>
          <li>How to manage your cookies</li>
          <li>Privacy policies of other websites</li>
          <li>Changes to our privacy policy</li>
          <li>How to contact us</li>
          <li>How to contact the appropriate authorities</li>
        </ul>

        <h1>What data do we collect?</h1>
        <p>Monkeytype collects the following data:</p>
        <ul>
          <li>Email</li>
          <li>Username</li>
          <li>Information about each typing test</li>
          <li>Your currently active settings</li>
          <li>How many typing tests you've started and completed</li>
          <li>How long you've been typing on the website</li>
        </ul>

        <h1>How do we collect your data?</h1>

        <p>
          You directly provide most of the data we collect. We collect data and
          process data when you:
        </p>
        <ul>
          <li>Create an account</li>
          <li>Complete a typing test</li>
          <li>Change settings on the website</li>
        </ul>

        <h1>How do we use your data?</h1>
        <p>Monkeytype collects your data so that we can:</p>

        <ul>
          <li>
            Allow you to view result history of previous tests you completed
            completed
          </li>
          <li>
            Save results from tests you take and show you statistics based on
            them
          </li>
          <li>Remember your settings</li>
          <li>Display leaderboards</li>
        </ul>

        <h1>How do we store your data?</h1>
        <p>Monkeytype securely stores your data using Firebase Firestore.</p>

        <h1>What are your data protection rights?</h1>
        <p>
          Monkeytype would like to make sure you are fully aware of all of your
          data protection rights. Every user is entitled to the following:
        </p>
        <ul>
          <li>
            The right to access – You have the right to request Monkeytype for
            copies of your personal data. We may limit the number of times this
            request can be made to depending on the size of the request.
          </li>
          <li>
            The right to rectification – You have the right to request that
            Monkeytype correct any information you believe is inaccurate. You
            also have the right to request Monkeytype to complete the
            information you believe is incomplete.
          </li>
          <li>
            The right to erasure – You have the right to request that Monkeytype
            erase your personal data, under certain conditions.
          </li>
          <li>
            The right to restrict processing – You have the right to request
            that Monkeytype restrict the processing of your personal data, under
            certain conditions.
          </li>
          <li>
            The right to object to processing – You have the right to object to
            Monkeytype processing of your personal data, under certain
            conditions.
          </li>
          <li>
            The right to data portability – You have the right to request that
            Monkeytype transfer the data that we have collected to another
            organization, or directly to you, under certain conditions.
          </li>
        </ul>
        <!-- <p>If you make a request, we have one month to respond to you. If you would like to exercise any of these rights, please contact us at our email: jack@monkeytype.com</p> -->

        <h1>Log Data</h1>
        <p>
          Like most websites, Monkeytype collects information that your browser
          sends whenever you visit the website. This data may include internet
          protocol (IP) addresses, browser type, Internet Service Provider
          (ISP), date and time stamp, referring/exit pages, and time spent on
          each page.
          <b>
            THIS DATA DOES NOT CONTAIN ANY PERSONALLY IDENTIFIABLE INFORMATION.
          </b>
          We use this information for analyzing trends, administering the site,
          tracking users' movement on the website, and gathering demographic
          information.
        </p>
        <p>In our case, this service is provided by Google Analytics.</p>

        <h1>Cookies</h1>
        <p>
          Cookies are text files placed on your computer to collect standard
          Internet log information and visitor behavior information. When you
          visit our websites, we may collect information from you automatically
          through cookies or similar technology
        </p>
        <p>
          For further information, visit
          <a href="https://www.allaboutcookies.org/">allaboutcookies.org</a>
          or
          <a href="https://en.wikipedia.org/wiki/HTTP_cookie">
            wikipedia.org/wiki/HTTP_cookie
          </a>
          .
        </p>

        <h1>How do we use cookies?</h1>
        <p>
          Monkeytype uses cookies in a range of ways to improve your experience
          on our website, including:
        </p>
        <ul>
          <li>Keeping you signed in</li>
          <li>Remembering your active settings</li>
          <li>Remembering your active tags</li>
        </ul>
        <h1>What types of cookies do we use?</h1>
        <p>
          There are a number of different types of cookies, however, our website
          uses:
        </p>
        <ul>
          <li>
            Functionality – Monkeytype uses these cookies so that we recognize
            you on our website and remember your previously selected settings.
          </li>
        </ul>
        <h1>How to manage cookies</h1>
        <p>
          You can set your browser not to accept cookies, and the above website
          tells you how to remove cookies from your browser. However, in a few
          cases, some of our website features may not function as a result or
          behave unexpectedly.
        </p>
        <h1>Privacy policies of other websites</h1>
        <p>
          Monkeytype contains links to other websites.
          <b>
            <u>
              Our privacy policy applies only to our website, so if you click on
              a link to another website, you should read their privacy policy.
            </u>
          </b>
        </p>

        <h1>Changes to our privacy policy</h1>
        <p>
          Monkeytype keeps its privacy policy under regular review and places
          any updates on this web page. The Monkeytype privacy policy may be
          subject to change at any given time without notice. This privacy
          policy was last updated on 20 January 2021.
        </p>
        <!-- TODO: add way to see when file was last committed to using the GitHub api-->

        <h1>How to contact us</h1>
        <p>
          If you have any questions about Monkeytypes’s privacy policy, the data
          we hold on you, or you would like to exercise one of your data
          protection rights, please do not hesitate to contact us.
        </p>
        <p>
          Email:
          <a href="mailto: bartnikjack@gmail.com">bartnikjack@gmail.com</a>
        </p>
        <p>Discord: Miodec#1512</p>
      </div>
    </div>
  </body>
</html>

```

**Comment by UnrealApex** [comment]
If possible, I will try to revert the commit, then redo the changes.

**Comment by UnrealApex** [comment]
I did a hard push, now only `privacy-policy.html` has been modified.

**Comment by Miodec** [comment]
Oh, im not exactly sure i agree with this change. I think I prefer `Table of Contents` more.

**Pull request merged**